TestWebService/Sources/TestWebService/TestWebService.swift

```diff
@@ -38,6 +38,8 @@ struct TestWebService: Apodini.WebService {
     }
     
     var configuration: Configuration {
+        HTTPConfiguration()
```

Comment:
Any idea how we could avoid this call and use the default anyway? Or is this fine?